Packs/CommonPlaybooks/Playbooks/playbook-Ticket_Management_-_Generic.yml

```diff
@@ -0,0 +1,703 @@
+      key:
+        simple: ZendeskTicketID
+      value:
+        complex:
+          root: alert
+          accessor: id
+          transformers:
+          - operator: If-Then-Else
+            args:
+              condition:
+                value:
+                  simple: lhs==rhs
+              conditionB: {}
+              conditionInBetween: {}
+              else:
+                value:
+                  simple: Zendesk.Ticket.id
+                iscontext: true
+              equals: {}
+              lhs:
+                value:
+                  simple: Zendesk.Ticket.id
+                iscontext: true
+              lhsB: {}
+              options: {}
+              optionsB: {}
+              rhs: {}
+              rhsB: {}
+              then: {}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 660,
+          "y": 530
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: a27c8fa0-c258-41db-84e9-8b93a26c1623
+    type: regular
+    task:
+      id: a27c8fa0-c258-41db-84e9-8b93a26c1623
+      version: -1
+      name: Set ServiceNow Ticket ID
+      description: commands.local.cmd.set.parent.incident.context
+      script: Builtin|||setParentIncidentContext
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "13"
+    scriptarguments:
+      key:
+        simple: ServiceNowTicketID
+      value:
+        complex:
+          root: alert
+          accessor: id
+          transformers:
+          - operator: If-Then-Else
+            args:
+              condition:
+                value:
+                  simple: lhs==rhs
+              conditionB: {}
+              conditionInBetween: {}
+              else:
+                value:
+                  simple: ServiceNow.Ticket.ID
+                iscontext: true
+              equals: {}
+              lhs:
+                value:
+                  simple: ServiceNow.Ticket.ID
+                iscontext: true
+              lhsB: {}
+              options: {}
+              optionsB: {}
+              rhs: {}
+              rhsB: {}
+              then: {}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 230,
+          "y": 530
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "13":
+    id: "13"
+    taskid: 4d9c9b8f-0747-4eb0-8474-99316ead7621
+    type: regular
+    task:
+      id: 4d9c9b8f-0747-4eb0-8474-99316ead7621
+      version: -1
+      name: 'Set Endpoint hostname from alerts '
+      description: commands.local.cmd.set.parent.incident.context
```

Comment:
Maybe add a more informational description explaining the if-then-else check.

Reply:
Done.